config/dev-portal.json

```diff
@@ -1,4 +1,7 @@
 {
   "$schema": "https://cdn.zuplo.com/schemas/dev-portal.json",
-  "enableAuthentication": true
+  "enableAuthentication": true,
+  "apiKeys": {
+    "bucketName": "z$env(ZUPLO_PROJECT_ID)-$env(ZUPLO_ENVIRONMENT_TYPE)"
```

Comment:
@AdrianMachado - why is there a prefix of `z` in front of the bucketname? Is that intentional?

Reply:
@vazexqi Yes, that is the format, see the linked template - it matches what we have on `core`